.github/workflows/main.yaml

```diff
@@ -66,11 +66,11 @@ jobs:
         with:
           submodules: true
       - name: Setup .NET
-        uses: actions/setup-dotnet@v1
+        uses: actions/setup-dotnet@v2
         with:
-          dotnet-version: 3.1.x
+          dotnet-version: 3.x
```

Comment:
with the 6.x tool chain you can build for all targets

Reply:
Ah, I bet I could remove the `--framework` argument. I'll give that a try.